packages/util/src/constants.ts

```diff
@@ -58,7 +58,7 @@ export const NetworkId: Record<Network, number> = {
 };
 
 export const STABLE_COINS = ['USDC', 'USDT', 'DAI', 'FRAX'];
-export const LSDS = ['wstETH', 'cbETH', 'rETH'];
+export const LSDS = ['wstETH', 'cbETH', 'rETH', 'sDAI'];
```

Comment:
Add sDAI to the list of stablecoins:
export const STABLE_COINS = ['USDC', 'USDT', 'DAI', 'FRAX', 'sDAI'];

Create a new list for showing the native token yield pop up:
export const NATIVE_YIELD = [...LSDS, 'sDAI']